test/helpers/generateToken.test.js

```diff
@@ -0,0 +1,23 @@
+import chai from 'chai';
+import generateToken from '../../server/helpers/generateToken';
+
+const { expect } = chai;
+
+describe('Test generateToken function', () => {
+  it('should return a token', async () => {
+    const payload = { id: 1 };
+    const token = await generateToken(payload, '24h');
```

Comment:
Since in the helpers this function for `generateToken` isn't asynchronous, is there any need to await it here in the test?